joblibs3/s3fs_backend.py

```diff
@@ -42,19 +51,17 @@ def _check_options(self, options):
     def configure(self, location, verbose=0,
                   backend_options=DEFAULT_BACKEND_OPTIONS):
         """Configure the store backend."""
-        compress = backend_options['compress']
-        options = self._check_options(backend_options.copy())
-
-        self.storage = s3fs.S3FileSystem(anon=options['anon'],
-                                         key=options['key'],
-                                         secret=options['secret'],
-                                         token=options['token'],
-                                         use_ssl=options['use_ssl'])
+        compress = backend_options.pop('compress', False)
+        # computation results can be stored compressed for faster I/O
+        self.compress = compress
```

Comment:
This could be simplified like the following I think:
```suggestion
        # computation results can be stored compressed for faster I/O
        self.compress = backend_options.pop('compress', False)
```